src/pages/Main/ProductsWrap.js

```diff
@@ -0,0 +1,227 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { PRODUCT_DATA } from './PRODUCT_DATA';
+import styled from 'styled-components';
+import { Navigation, Pagination, Scrollbar, A11y } from 'swiper';
+
+import { Swiper, SwiperSlide } from 'swiper/react';
+
+import 'swiper/css';
+import 'swiper/css/navigation';
+import 'swiper/css/pagination';
+import 'swiper/css/scrollbar';
+
+const ProductsWrap = () => {
+  const AssmentComponent = ({ url, name }) => {
+    return (
+      <AssessmentIndividual>
+        <AssessmentIcon src={'/images/Search/' + url} />
+        <AssessmentText>{name}</AssessmentText>
+      </AssessmentIndividual>
+    );
+  };
+  return (
+    <>
+      <SoldOutSoon>매진임박! 지금 신청하세요!</SoldOutSoon>
+      <Swiper
+        modules={[Navigation, Pagination, Scrollbar, A11y]}
+        spaceBetween={100}
+        slidesPerView={4}
+        navigation
+      >
+        <ProductWrap>
+          {PRODUCT_DATA?.map(
+            ({
+              id,
+              route_url,
+              img_url,
+              coupon,
+              user_id,
+              class_title,
+              sale,
+              class_heart,
+              class_like,
+              price,
+              term,
+            }) => (
```

Comment:
* PRODUCT_DATA에 있는 데이터를 다 쓸거라면, 인자로 한번에 넘겨주는게 코드 가독성이 좋습니다.
* 구조분해는 인자로 받아서 해주세요.
```suggestion
            PRODUCT_DATA.map(productData,index) => (
```